Templates/AppSource App/.github/workflows/_BuildALGoProject.yaml

```diff
@@ -74,7 +74,7 @@ env:
 jobs:
   BuildALGoProject:
     runs-on: ${{ fromJson(inputs.runsOn) }}
-    name: ${{ inputs.project }} - ${{ inputs.buildMode }}
+    name: ${{ inputs.project }} (${{ inputs.buildMode }})
```

Comment:
Missing ProjectName input in the AppSource yaml  (and this should reference inputs.projectName)

Reply:
Else it looks good to me.

Reply:
Oh, nice catch!